build.gradle

```diff
@@ -0,0 +1,154 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3"
+    }
+}
+
+plugins {
+    id 'checkstyle'
+    id 'io.spring.dependency-management' version '1.0.3.RELEASE'
+    id 'org.springframework.boot' version '1.5.8.RELEASE'
+}
+
+apply plugin: 'checkstyle'
+apply plugin: 'java-library'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'maven'
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.bintray'
+
+group 'uk.gov.hmcts.reform'
```

Comment:
mismatching with file structure: is it going under platform reform group or just generic hmcts authorisation package?

Reply:
reworded file structure